tasks/lib/compass.js

```diff
@@ -169,11 +167,35 @@ exports.init = function (grunt) {
       }
     }
 
+    // add converted options
+    [].push.apply(args, dargs(options, [
+      'raw',
+      'clean',
+      'bundleExec',
+      'basePath',
+      'specify',
+      'watch'
+    ]));
+
+    if (grunt.option('no-color')) {
+      args.push('--boring');
+    }
+
+    var doubleDash = false;
+    function pushDoubleDash() {
+      if (!doubleDash) {
+        args.push('--');
+        doubleDash = true;
+      }
+    }
```

Comment:
Would be cleaner to wrap this function in [`onetime`](https://github.com/sindresorhus/onetime).